docs/operations/rule-configuration.md

```diff
@@ -23,212 +23,242 @@ title: "Retaining or automatically dropping data"
   "tieredReplicants": {
     "hot": 1,
-    "_default_tier" : 1
+    "_default_tier": 1
   }
 }
 ```
 
-* `type` - this should always be "loadByInterval"
-* `interval` - A JSON Object representing ISO-8601 Intervals
-* `tieredReplicants` - A JSON Object where the keys are the tier names and values are the number of replicas for that tier.
+Set the following properties:
+- `interval`: a JSON object representing [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) intervals.
+- `tieredReplicants`: a JSON object containing tier names and the number of replicas for each tier.
 
-### Period Load Rule
+### Period load rule
 
-Period load rules are of the form:
+Period load rules have type `loadByPeriod` and the following example API payload:
 
 ```json
 {
-  "type" : "loadByPeriod",
-  "period" : "P1M",
-  "includeFuture" : true,
+  "type": "loadByPeriod",
+  "period": "P1M",
+  "includeFuture": true,
   "tieredReplicants": {
       "hot": 1,
-      "_default_tier" : 1
+      "_default_tier": 1
   }
 }
 ```
 
-* `type` - this should always be "loadByPeriod"
-* `period` - A JSON Object representing ISO-8601 Periods
-* `includeFuture` - A JSON Boolean indicating whether the load period should include the future. This property is optional, Default is true.
-* `tieredReplicants` - A JSON Object where the keys are the tier names and values are the number of replicas for that tier.
+Set the following properties:
+- `period`: a JSON object representing [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) periods. The period is from some time in the past to the future or to the current time, depending on the `includeFuture` flag.
+- `includeFuture`: a boolean flag to indicate whether the load period includes the future. Defaults to `true`.
+- `tieredReplicants`: a JSON object containing tier names and the number of replicas for each tier.
 
-The interval of a segment will be compared against the specified period. The period is from some time in the past to the future or to the current time, which depends on `includeFuture` is true or false. The rule matches if the period *overlaps* the interval.
+Druid compares a segment's interval to the period you specify in the rule. The rule matches if the period overlaps the segment interval. 
 
-## Drop Rules
+## Drop rules
 
-Drop rules indicate when segments should be dropped from the cluster.
+Drop rules define when Druid drops segments from the cluster.
 
-### Forever Drop Rule
+### Forever drop rule
 
-Forever drop rules are of the form:
+Forever drop rules have type `dropForever`:
 
 ```json
 {
-  "type" : "dropForever"
+  "type": "dropForever"
 }
 ```
 
-* `type` - this should always be "dropForever"
+Druid drops all segments with this rule from the cluster. 
 
-All segments that match this rule are dropped from the cluster.
+### Interval drop rule
 
-
-### Interval Drop Rule
-
-Interval drop rules are of the form:
+Interval drop rules have type `dropByInterval` and the following example API payload:
```

Comment:
Usual request to explain what this does. I'm going to guess.

> We noted above that you may have extra activity on the most recent data, and thus want to load extra replicas on a dedicated tier. You may also find that older data is almost never used, and it is not worth that data taking space on the cluster, yet you want to retain the data for the occasional request, or for compliance reasons.
> 
> Use a drop interval rule to prevent a range of data from loading onto _any_ tier. The range is typically your oldest data. This data will reside in cold storage, but will not be queryable. If you need to query the data, simply remove (or modify) the drop role so that the needed data is again loaded.

Reply:
Updated with the second part of this explanation. The doc is more of a reference than for a thorough explanation of use cases, which I think do not fit here.